pkg/container-collection/containers.go

```diff
@@ -167,3 +177,15 @@ func GetColumns() *columns.Columns[Container] {
 func (c *Container) IsEnriched() bool {
 	return c.Name != "" && c.Podname != "" && c.Namespace != "" && c.PodUID != "" && c.Runtime != ""
 }
+
+func GetContainerEventColumns() *columns.Columns[ContainerEvent] {
+	return columns.MustCreateColumns[ContainerEvent]()
+}
+
+func ContainerEventFromContainer(container Container, event, timestamp string) *ContainerEvent {
+	return &ContainerEvent{
+		Container: container,
+		Timestamp: timestamp,
+		Event:     event,
+	}
+}
```

Comment:
I think this function is too specific to being added to the package. The callers could implement it.

Reply:
I wasn't sure about this as well so I agree it should be on the client side. I can move this to `cmd/local-gadget/containers`